custom/conf/app.example.ini

```diff
@@ -878,6 +878,9 @@ ROUTER = console
 ;; Allow deletion of unadopted repositories
 ;ALLOW_DELETION_OF_UNADOPTED_REPOSITORIES = false
 
+;; Dont' allow download source tar files from UI
+;DISABLED_DOWNLOAD_SOURCE = false
```

Comment:
Please also add to cheat sheet docs

Reply:
I think you should mention the source *archive* not just source

Reply:
small typo - `Dont'`